shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/segment/ddl/column/position/ColumnAfterPositionSegment.java

```diff
@@ -27,7 +28,7 @@
     
     private final String afterColumnName;
```

Comment:
Why do not change it as ColumnSegment?

Reply:
ColumnPositionSegment include columnName, and the type of columnName is ColumnSegment

Reply:
ColumnPositionSegment include columnName, the type od columnName is ColumnSegment.